public/js/main.js

```diff
@@ -33,7 +33,7 @@ $("#contact-form").on("submit", function (e) {
   }
   var payload = { email: email, name: email, website: email, comment: 'paywall.fpjs.dev' };
   $.ajax({
```

Comment:
we don't need this, there should be just a link to the dashboard, we don't need to submit leads from the paywall @kalexey89

Reply:
Oh, yes, I see. Removed this code.